### ISSUE TITLE
m0 adafruit/arduino boards problem

### ISSUE THREAD

**Issue by brorgustav** [adafruit/FifteenStep#7]
I cannot seem to get the >new< version library to work with M0 (it doesnt state anywhere if its supported or not). It seems strange since the last version works alright. Also the sequencer seems to send the note to the callback before passing the amount of max steps. But it only happens sometimes, which made me think it could be the quantization?

**Comment by ladyada** [comment]
yeah this code is quite old and not maintained, if you know where the error is and you can submit a PR, it'd be welcome!
